### ISSUE TITLE
Crashing when overriding a property name in reference

### ISSUE THREAD

**Issue by pksunkara** [apiaryio/drafter#118]
The following blueprint when parsing with drafter crashes with the message given at the end of the post.

```apib
# GET /
+ Response 200 (application/json)
  + Attributes (A)
      + c: 42

# Data Structures
## A (object)
+ c: 2
```

```
pksunkara@apicave → ./bin/drafter a.apib 
libc++abi.dylib: terminating with uncaught exception of type char const*
Abort trap: 6
```

**Comment by klokane** [comment]
I try it on **master*** it is parsed on my computer:
```
% ./bin/drafter crash.apib                                                                                                                                [klok@caterpillar:~/projects/apiary/drafter]
element: "category"
meta:
  classes:
    - "api"
  title: ""
attributes:
  meta:
    -
      element: "member"
      meta:
        classes:
          - "user"
      content:
        key:
          element: "string"
          content: "+ c"
        value:
          element: "string"
          content: "42"
content:
  -
    element: "category"
    meta:
      classes:
        - "dataStructures"
    content:
      -
        element: "dataStructure"
        content:
          -
            element: "object"
            meta:
              id: "A"
              description: "          + c: 2\n"

OK.
warning: (3)  ignoring possible metadata, expected '<key> : <value>', one one per line :0:73
```

**Comment by klokane** [comment]
@pksunkara have you updated submodules?

**Comment by zdne** [comment]
@klokane with the latest drafter (master) I am getting the crash too.

```
yt:Desktop z$ drafter --format=json mson.apib > out.json
libc++abi.dylib: terminating with uncaught exception of type char const*
Abort trap: 6
```

**Comment by zdne** [comment]
<img width="1254" alt="screen shot 2015-09-15 at 3 16 13 pm" src="https://cloud.githubusercontent.com/assets/613617/9877371/bfe84f7c-5bbc-11e5-9ab8-e7654c278949.png">

**Comment by klokane** [comment]
Sry it crash, I didn't paste full blueprint. It crash in libsos while serialization because of key duplicity.

**Comment by pksunkara** [comment]
Fixed by #120
